bitcoin/src/blockdata/transaction.rs

```diff
@@ -1173,6 +1171,30 @@ impl From<&Transaction> for Wtxid {
     fn from(tx: &Transaction) -> Wtxid { tx.wtxid() }
 }
 
+/// Returns the effective_value `Amount` returning `None` on overflow.
+///
+/// The effective value is the value of this output minus the amount to spend it.  That is, the
+/// effective_value can be calculated as value - (fee_rate * weight).
+///
+/// Note: the effective value of a [`Transaction`] may increase more than the effective value of
```

Comment:
Less, not more.

Reply:
good catch.  Since the transaction size increases more, the effective value is less.

Reply:
resolved